common/build/eslint-config-fluid/rushstack.js

```diff
@@ -33,12 +32,9 @@ module.exports = {
         "ecmaVersion": 2018,
         "sourceType": "module",
         "project": "./tsconfig.json",
-        // "tsconfigRootDir": __dirname
     },
     "plugins": [
-        // "@typescript-eslint",
         "no-null",
-        // "optimize-regex",
         "prefer-arrow",
```

Comment:
not totally relevant, but i dislike this rule, i have to suppress it all the time as it makes code unreadable. I was thinking about changing to https://eslint.org/docs/3.0.0/rules/prefer-arrow-callback which seems like a better balance. 

nothing you need to do here, but if you want you could update this as well, otherwise i'm sure i'll do it eventually

Reply:
I included this change.